model/cloud.go

```diff
@@ -52,6 +52,7 @@ type Product struct {
 	Family            SubscriptionFamily `json:"product_family"`
 	RecurringInterval RecurringInterval  `json:"recurring_interval"`
 	BillingScheme     BillingScheme      `json:"billing_scheme"`
+	CrossSellsTo      string             `json:"cross_sells_to"`
```

Comment:
I think we'll also want to add CrossSellsTo to `UserFacingProduct`, a simplified version of the same data that we use in some cases to display product related things to non-admin users. I attached a video of an example of when non-admin users see the pricing modal. I'm imagining we'll also want to let them see the annual vs monthly toggle that admin users would be able to see, so I think we'd also need the `CrossSellsTo` attribute for that.

https://user-images.githubusercontent.com/13738432/196480265-dd89ff07-5aac-496d-ba7e-09fe9fa61134.mp4

Reply:
Good question! Yes, I think `RecurringInterval` should also be added. I don't see others that need to be added at this time.

Reply:
Great. thanks :)